### PR TITLE
Feature/136 integrate with login

### PR DESCRIPTION
**Pull request by dami-gg**
Preparation for #136 but since I had to do big changes let's merge this to master.

Since I removed the mocks, for any future test we need to temporarily add the USER_6 and mock the environment like we used to do in the past

### CODE REVIEW

File: src/app/components/products/products.html [hunk 1]
```diff
@@ -6,16 +6,16 @@
          ng-class="{'highlighted': $ctrl.isHighlighted(product)}"
          ng-click="$ctrl.switchMenu(product.id)"
          href>
-        <span class="">{{product.name}}</span>
+        <span class="">{{::product.name}}</span>
```
Comment (frontendherodk) [inline src/app/components/products/products.html]: Good catch!

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Comment by dami-gg** [comment]
@frontendherodk there was a small conflict, feel free to merge this when you're back and use it as base for next issues

**Comment by dami-gg** [comment]
:+1:

**Comment by frontendherodk** [comment]
👍

**Pull request merged**